atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/asts/DefaultSweepStateCoordinator.java

```diff
@@ -34,32 +38,46 @@
 import org.immutables.value.Value;
 
 public final class DefaultSweepStateCoordinator implements SweepStateCoordinator {
+    private static final SafeLogger log = SafeLoggerFactory.get(DefaultSweepStateCoordinator.class);
     private final LockableFactory<SweepableBucket> lockableFactory;
     private final CandidateSweepableBucketRetriever candidateSweepableBucketRetriever;
 
     // Exists to facilitate testing, and to make switching out the shuffle strategy easier.
     private final Function<List<Lockable<SweepableBucket>>, Stream<Lockable<SweepableBucket>>> iterationOrderGenerator;
 
-    private volatile Set<Lockable<SweepableBucket>> seenBuckets;
-    private volatile BucketsLists bucketsLists;
+    private volatile Set<Lockable<SweepableBucket>> seenBuckets = ConcurrentHashMap.newKeySet();
```

Comment:
addresses an NPE that the sweepers would hit when trying to run before the first load. It didn't really matter since they'd just retry, but it was unnecessary noise.